src/gg.regression.unity.bots/Runtime/Scripts/RGLegacyInputUtility/RGLegacyInputWrapper.cs

```diff
@@ -431,6 +433,17 @@ private static void SendMouseEventMessage(GameObject gameObject, string methodNa
             }
         }
 
+        private static void SendPointerEvent(GameObject gameObject, string methodName)
```

Comment:
This is the code change that actually makes pointer events work in our legacy input code